Editorial pass (#26)
intarchboard/draft-protocol-maintenance: draft-iab-protocol-maintenance.md

For example, in TLS {{?TLS=RFC8446}}, extensions use a tag-length-value format
and they can be added to messages in any order.  However, some server

> implementations terminated connections if they encounter a TLS ClientHello

Reviewer: I'll you decide since you're the native speaker but shouldn't the two tenses agree?
```suggestion
implementations terminated connections if they encountered a TLS ClientHello
```

the receiving party is being tolerant of unexpected inputs.

A well-designed extensibility mechanism establishes clear rules for the handling

> of things like new messages or parameters.  This depends on having precise
> specification for handling of malformed or illegal inputs so that

Reviewer: Again up to you but this sounds weird to me because I think of a "specification" as a whole document. How about:
```suggestion
of things like new messages or parameters.  This depends on precisely
specifying the handling of malformed or illegal inputs so that
```
